taxonomy/migrations/0027_auto_20221107_0342.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 3.2.15 on 2022-11-07 03:42
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+import model_utils.fields
+
+
+class Migration(migrations.Migration):
```

Comment:
What do you think about generating a named migration vs relying on auto generation? IMHO named migration gives more idea about what it is about.